routing/car_router.cpp

```diff
@@ -196,41 +197,86 @@ class OSRMRoutingResult : public turns::IRoutingResult
   RoutingMapping & m_routingMapping;
 };
 
+bool FindSingleOsrmRoute(FeatureGraphNode const & source, FeatureGraphNode const & target,
+                         RouterDelegate const & delegate, TRoutingMappingPtr & mapping, Index const & index,
+                         Route & mwmRoute)
+{
+  vector<Junction> mwmRouteGeometry;
```

Comment:
Yes. And the word Route in mwmRouteGeometry is not helpful as well. I removed all these mwm in the method. and renamed mwmRouteGeometry to geometry.